packages/pods-core/src/builtin/MarkdownPod.ts

```diff
@@ -336,6 +333,21 @@ export class MarkdownExportPod extends ExportPod {
         return fs.writeFile(fpath, body);
       })
     );
+
+    // Export Assets
+    vaults.forEach(async (vault) => {
+      //TODO: Avoid hardcoding of assets directory, or else extract to global const
+      const destPath = path.join(dest.fsPath, vault.fsPath, "assets");
+      const srcPath = path.join(wsRoot, vault.fsPath, "assets");
```

Comment:
we have multiple types of [vaults](https://wiki.dendron.so/notes/6682fca0-65ed-402c-8634-94cd51463cc4.html#vault-types). depending on the type, the path might be different. you should use [`getRelPath`](https://github.com/dendronhq/dendron/blob/md-pod-export/packages/common-all/src/vault.ts#L32:L32) to get relative path to the vault from the workspace

Reply:
Hmm, I tried making this change, but then it's causing some issues in my tests.

What is getRelPath intended to return?  Maybe the same question, but when is vault.workspace meant to be set? Is it supposed to be set when a vault is added to a workspace? I notice that this value is not set in the test environment.  Is this intentional, or is this a bug? 

If I do set a value for vault.workspace in the test, then this conflicts with the following logic in DNote Utils, which will return a double prefix (something like `C:\mydir\C:\mydir\vault1`)
```typescript
  static getFullPath(opts: {
    wsRoot: string;
    vault: DVault;
    basename: string;
  }) {
    const root = path.isAbsolute(opts.vault.fsPath)
      ? VaultUtils.getRelPath(opts.vault)
      : path.join(opts.wsRoot, VaultUtils.getRelPath(opts.vault));
    return path.join(root, opts.basename);
  }
```

Reply:
discussed offline, let me know if you need more context here